UT4MasterServer/Models/Requests/GrantXP.cs

```diff
@@ -0,0 +1,10 @@
+using System.Text.Json.Serialization;
+
+namespace UT4MasterServer.Models.Requests
```

Comment:
Use file-scoped namespaces to avoid additional tabbing out.

Reply:
Done